### ISSUE TITLE
Frequency tracking

### ISSUE THREAD

**Issue by BPaton** [CMRR-C2P/Spectroscopy#1]
Hi There,

We have been having some temperature (of the gradients/system) related frequency drift issues with PRESS and other spectro sequences on our VD-13C Skyra and one "fix" that Siemens have helped us with is a new Breast spectro sequence that has frequency tracking. After reading over the many great features in your spectro package I was hoping to find out if it also has some kind of frequency tracking especially for MEGA-PRESS?

Thanks.

Bryan.

**Comment by eauerbach** [comment]
No, we have not done this. Since we usually save the individual averages and frequency correct them in postprocessing it hasn't been necessary (I don't think Siemens provides this option in the product sequences). We haven't run into cases where the frequency has drifted so much during a scan that frequency selective pulses used for water suppression or editing have failed, where you would really need a realtime correction...but maybe this is an issue on Skyra, I don't know.

**Comment by BPaton** [comment]
Hi Eddie, Thanks for the information. There is a brand new Siemens product PRESS sequence used for Breast spectro that explicitly does frequency tracking and we have recently started using it due to the frequency drift issues. After bringing this issue to the attention of Siemens a number of remedies were offered. During all of our spectro sequences, product and otherwise we get varying degrees of peak shift, which when averaged result in peak broadening. The shift is sometimes bad enough that you can observe it by looking at individual averages as they come in. 

The drift itself is caused by a number of factors on our Skyra. Any heating of the gradients by sequences that might push things, EPI BOLD, Diffusion etc heats the coils enough to observe frequency drift over the next few hours depending on how much heating originally occurred. So doing spectro first thing in the morning is a simple but inconvenient fix for this. Also our Skyra has an ECO mode it goes into overnight where the temperature point is slightly elevated to 24 rather than 23 degrees C to conserve power. Apparently this feature was added in our last update but we did not know anything about it. :) Unless this ECO mode is disabled the morning scanning will also have a temperature ramp down which further adds to the drift.

**Comment by gbook** [comment]
Hi BPaton,
We also have a Skyra and they are planning to upgrade us to VE11C, which I believe contains this ECO mode. Is there a way to disable the ECO mode?
-Greg

**Comment by BPaton** [comment]
There is a way to disable it but you will need to speak to your Siemens engineers for them to do this.
